From a0a9c89e9a7f5b19ed3705d4aab099e68eaa247f Mon Sep 17 00:00:00 2001
Message-Id: <a0a9c89e9a7f5b19ed3705d4aab099e68eaa247f.1684964119.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684964119.git.sharlatanus@gmail.com>
References: <cover.1684964119.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 May 2023 22:48:32 +0100
Subject: [PATCH v2 02/20] gnu: python-asdf-wcs-schemas: Symplify.

* gnu/packages/astronomy.scm (python-asdf-wcs-schemas): Symplify package.
[build-system]: Use pyproject-build-system.
---
 gnu/packages/astronomy.scm | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index e180a2f81a..0d0e45a3b1 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3020,19 +3020,9 @@ (define python-asdf-wcs-schemas
        (uri (pypi-uri "asdf_wcs_schemas" version))
        (sha256
         (base32 "0khyab9mnf2lv755as8kwhk3lqqpd3f4291ny3b9yp3ik86fzhz1"))))
-    (build-system python-build-system)
-    (arguments
-     `(#:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda* (#:key inputs outputs tests? #:allow-other-keys)
-             (when tests?
-               (add-installed-pythonpath inputs outputs)
-               (invoke "python" "-m" "pytest")))))))
+    (build-system pyproject-build-system)
     (native-inputs
-     (list python-pytest
-           python-setuptools-scm
-           python-semantic-version))
+     (list python-pytest python-setuptools-scm python-semantic-version))
     (propagated-inputs
      (list python-asdf))
     (home-page "https://github.com/asdf-format/asdf-wcs-schemas")
-- 
2.39.2

